src/screens/ChoosePlanScreen/ChoosePlan.tsx

```diff
@@ -424,6 +421,10 @@ function ChoosePlan() {
                 </Box>
               ))}
             </Box>
+            {/* {items[currentPosition].name !== 'Pleb' ?
```

Comment:
remove commented code lines if not being used.

Reply:
Removed.